src/scs/pki.md

```diff
@@ -37,3 +37,9 @@ In a two-tier hierarchy the Root CA and issuing (Subordinate) CAs are separated
 
 In a three-tier CA an intermediate CA is placed between the Root CA and the Subordinate (issuing) CA. This is done to further separate the Root CA from low-level CA operations. The middle layer (intermediate CA) is only used to sign Subordinate CAs that issue the end-entity certificates. 
```

Comment:
This is done to separate the Root CA from low-level CA operations.
(the word further must be deleted)